InvenTree/InvenTree/models.py

```diff
@@ -73,19 +72,16 @@ def getUniqueChildren(self, unique=None):
 
         unique.add(self.id)
 
+        # commenting this out as it throws errors.
```

Comment:
Well, for flake8 in the current setting, unused code segments are considered an error. As you did not make use of `children` yet, it is considered unused. When I comment it out, `contents` will become unused too. You are very welcome to re-include the code, but it would be preferable if you'd then make use of `children` so it isn't just dead code.

The actual `make style` output was:
`./InvenTree/InvenTree/models.py:77:9: F841 local variable 'children' is assigned to but never used`